src/beautifiers/clang-format.coffee

```diff
@@ -0,0 +1,60 @@
+###
+Requires clang-format (https://clang.llvm.org)
+###
+
+"use strict"
+Beautifier = require('./beautifier')
+path = require('path')
+fs = require('fs')
+
+module.exports = class ClangFormat extends Beautifier
+
+  name: "clang-format"
+
+  options: {
+    "C++": true
+    "C": true
+    "Objective-C": true
```

Comment:
It does not look like any of the `options` are used, so change it from `true` to `false` to indicate that while the language is supported it does not support any of the options directly.